src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -213,7 +213,25 @@ public static A11yElement GetAppElement(A11yElement e)
         }
 
         /// <summary>
-        /// Get IUIAutomationTreeWalker based on inidcated mode.
+        /// Normalize an element to the specified TreeViewMode
+        /// </summary>
+        /// <param name="element">A11yElement</param>
+        /// <param name="treeViewMode">mode to normalize to</param>
+        /// <returns></returns>
+        public static A11yElement GetNormalizedElement(A11yElement element, TreeViewMode treeViewMode)
```

Comment:
I think the word "normalized" should be removed. passing in the tree scope communicates well the purpose of the function and normalized doesn't carry much meaning in this context.

Reply:
We discussed this and didn't find a better alternative, so leaving it as is...